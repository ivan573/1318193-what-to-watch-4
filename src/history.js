import {createBrowserHistory} from "history";

const history = createBrowserHistory();

export {history as default};
