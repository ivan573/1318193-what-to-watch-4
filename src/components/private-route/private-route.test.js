import React from "react";
import renderer from "react-test-renderer";
import {PrivateRoute} from "./private-route.jsx";
import {Router} from "react-router-dom";
import history from "../../history.js";

test(`Private route renders without errors`, () => {
  const tree = renderer
    .create(<Router history={history}>
      <PrivateRoute
        authorizationStatus={`NO_AUTH`}
        exact={true}
        path={`/mylist`}
        render={() => {
          return <div></div>;
        }}
      />
    </Router>).toJSON();

  expect(tree).toMatchSnapshot();
});
