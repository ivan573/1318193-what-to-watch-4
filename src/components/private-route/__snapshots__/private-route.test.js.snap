// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Private route renders without errors 1`] = `null`;
